force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js

```diff
@@ -0,0 +1,164 @@
+import { createElement } from 'lwc';
+import BrokerCard from 'c/brokerCard';
+import { getNavigateCalledWith } from 'lightning/navigation';
+import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+
+// Mock realistic data
+const mockGetPropertyRecord = require('./data/getPropertyRecord.json');
+
+// Register as an LDS wire adapter. Some tests verify the provisioned values trigger desired behavior.
+const getRecordAdapter = registerLdsTestWireAdapter(getRecord);
+
+const BROKER_ID = 'a003h000003xlBiAAI';
+
+const BROKER_FIELDS_INPUT = [
+    {
+        fieldApiName: 'Name',
+        objectApiName: 'Broker__c'
+    },
+    {
+        fieldApiName: 'Phone__c',
+        objectApiName: 'Broker__c'
+    },
+    {
+        fieldApiName: 'Mobile_Phone__c',
+        objectApiName: 'Broker__c'
+    },
+    {
+        fieldApiName: 'Email__c',
+        objectApiName: 'Broker__c'
+    }
+];
+
+const OBJECT_API_NAME_INPUT = 'Broker__c';
+
+describe('c-broker-card', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    // Helper function to wait until the microtask queue is empty. This is needed for promise
+    // timing.
+    function flushPromises() {
+        // eslint-disable-next-line no-undef
+        return new Promise((resolve) => setImmediate(resolve));
+    }
+
+    describe('broker record form', () => {
+        it('gets property data from wire service', () => {
+            // Create element
+            const element = createElement('c-broker-card', {
+                is: BrokerCard
+            });
+            document.body.appendChild(element);
+
+            // Emit data from @wire
+            getRecordAdapter.emit(mockGetPropertyRecord);
+
+            return flushPromises().then(() => {
+                const propertyEl = element.shadowRoot.querySelector(
+                    'lightning-record-form'
+                );
+                expect(getFieldValue.mock.calls.length).toBe(1);
```

Comment:
You can use `expect(getFieldValue).toHaveBeenCalled();`

Reply:
made the changes as suggested